python/python/lance/dataset.py

```diff
@@ -1448,6 +1448,7 @@ def create_index(
         precomputed_partition_dataset: Optional[str] = None,
         storage_options: Optional[Dict[str, str]] = None,
         filter_nan: bool = True,
+        one_pass_ivfpq: bool = False,
```

Comment:
let's just make it the default.

Reply:
I would rather we don't, since it is incompatible with local pq. I'm not strongly against making it default though.